src/test/scala/com/itechart/ny_accidents/utils/DateUtilsTest.scala

```diff
@@ -1,17 +1,33 @@
 package com.itechart.ny_accidents.utils
 
-import com.itechart.ny_accidents.constants.GeneralConstants
+import java.time.format.DateTimeFormatter
+
 import org.scalatest.FunSpec
 
 class DateUtilsTest extends FunSpec {
   // Will be rewrite. Test failed on different timezone machines
   // TODO need fix
   describe("DateUtils") {
-//    it("should return valid value") {
-//      val expectedResult = 1558095780000L
-//      val testValue = "17.05.2019 15:23"
-//
-//      assert(DateUtils.parseDateToMillis(testValue, GeneralConstants.DATE_TIME_WEATHER_PATTERN).contains(expectedResult))
-//    }
+    //    it("should return valid value") {
+    //      val format: DateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm")
+    //      val expectedResult = 1558095780000L
+    //      val testValue = "17.05.2019 15:23"
+    //
+    //      assert(DateUtils.parseDateToMillis(testValue, format).contains(expectedResult))
+    //    }
+
+    it("should add one hour to date as long") {
+      val testValue = 1558095780000L
+      val expectedValue = 1558099380000L
+
+      assert(DateUtils.addHour(testValue) == expectedValue)
```

Comment:
What's the purpose of this test? 
These computations should be based on functionality, provided by JDK out of the box (LocalDateTime.plusHours)

Reply:
We can't compute it using LocalDateTime, because out method works with raw Long parameters.

`def subtractHour(dateTimeMillis: Long): Long = {
    dateTimeMillis - MILLIS_IN_HOUR
  }

  def addHour(dateTimeMillis: Long): Long = {
    dateTimeMillis + MILLIS_IN_HOUR
  }`